CHANGES.txt

```diff
@@ -1,6 +1,147 @@
 Traits UI Changelog
 ===================
 
+Release 7.2.0
+-------------
+TraitsUI 7.2.0 is a minor release which includes numerous bug fixes,
+documentation improvements, code maintenance changes, and enhancements.
+
+Highlights of this release
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* The migration from ``on_trait_change`` to ``observe`` is underway. As a
+  result, TraitsUI now requires Traits >= 6.1.
+* New display-only VideoEditor (currently only on Qt backend).
+* Exapnsion of features for the new ``UITester`` including the ability to
+  inspect UI object visibility / enabledness. Also documentation for testing
+  has been updated.
+
+Notes on upgrading
+~~~~~~~~~~~~~~~~~~
+
+* This release of TraitsUI now depends on Traits 6.1+. Also, deprecated code /
+  modules have been removed. Namely, the :mod:`traitsui.image <traitsui.image>`
+  module which was moved to ``pyface.image``, ``editors_gen`` modules, Editor
+  and EditorFactory factory methods on Toolkit objects, and more.  For a
+  complete list, see PRs in the "Removals" section below.  These were all
+  generally unused / deprecated for sometime. Also, importing directly from
+  ``traitsui.editors`` has been deprecated. Please update imports to import
+  directly from :mod:`traitsui.api` or :mod:`traitsui.editors.api`.
```

Comment:
Which should we be recommending here? I listed both currently, but I imagine we just want people to import from `traitsui.api`?  
This has been discussed before

Reply:
>  This has been discussed before

I dont remember that discussion - or the conclusion of that discussion.
I'm fine with listing both. I completely forgot about the fact that the editor factories are also importable via `traitsui.api`. I think we can leave it up to the users to choose. I don't think we'll remove `traitsui.editors.api`.